devel/ld64/Portfile

```diff
@@ -78,6 +78,15 @@ subport ld64-97 {
         ld64-97-no-ppc-thread_status.patch \
         ld64-97-configured_architectures.patch
 
+    variant tigerrpath {
```

Comment:
The logic should be reverse here. We do not need this variant outside of 10.4, and certainly a variant which just does nothing.